dracut.sh

```diff
@@ -822,6 +825,7 @@ while :; do
             ;;
         --keep) keep="yes" ;;
         --printsize) printsize="yes" ;;
+        --keep-cryptkeyfiles)        keep_cryptkeyfiles="yes";;
```

Comment:
You need to export `keep_cryptkeyfiles` here:

https://github.com/openSUSE/dracut/blob/67527a99f1c5cad482d7bb78e68b5437a44b9027/dracut.sh#L1920-L1939

Reply:
Ok, thanks!